### PR TITLE
added comment

### PR DESCRIPTION
**Pull request by TeeniJai**
I have no idea what the other change is, hopefully nothing bad ;-;

### CODE REVIEW

File: hi.java [hunk 1]
```diff
@@ -4,7 +4,7 @@ public static void main(String args[]){
   
       System.out.println("Hi");
   
-  
+  //Hey, I hope this works! :3 //
```
Comment (CatarinaK1) [inline hi.java:7]: Doesnt work but thats ok :3

**Review by CatarinaK1 — APPROVED** [review]
ok

File: hi.java
Comment (KoblaQ) [inline hi.java:1]: I see what you did here.

File: .project
Comment (KoblaQ) [inline .project:1]: I have no idea what's going on here. lol

**Review by KoblaQ — APPROVED** [review]
Everything works. Thanks.

**Pull request merged**